docs/src/appendix/cast/account/add.md

```diff
@@ -4,7 +4,7 @@ Import an account to accounts file.
 Account information will be saved to the file specified by `--accounts-file` argument,
 which is `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default.
 
-## Required common arguments - passed by CLI or specified in Scarb.toml
+## Required Common Arguments - Passed by CLI or Specified in Scarb.toml
```

Comment:
Not sure about capitalizing the part after `-`, this is not really a part of a title